src/containers/Application.js

```diff
@@ -124,11 +165,14 @@ class Application extends Component {
 		else return (
 			<Provider store={this.Store}>
 				<div className="app">
-					<Header app={this}/>
+					<Header hasSidebar={this.props.hasSidebar} app={this}/>
 					<div className="app-body">
-						<Sidebar app={this} navigation={this.Navigation}/>
+						{(this.props.hasSidebar || typeof this.props.hasSidebar === 'undefined') ? 
+							<Sidebar app={this} navigation={this.Navigation} display="lg"/> : 
+							<Sidebar app={this} navigation={this.Navigation} display="xs"/>}
```

Comment:
You still want to display Sidebar when `this.props.hasSidebar` is `False`.

Reply:
Yes. When False, it will display Sidebar and items in it only if there are any items added to the Header, otherwise it will not display the sidebar. It basically resolves the Header issue with collapsing the Header items / components into a toggler button when you are on smaller screens. 
Let say, that Header contains e.g. 2 buttons, <button1/> and <button2/>. If you want to use them and collapse them, when the screen is diminished, you can add them as items into the Navigation of the Sidebar, even though the Sidebar is "disabled" by False. Then, when the screen change the size (e.g. when you diminish the browser screen or you are on your mobile phone), the items from the Header disappear and SidebarToggler button will appear with the <button1/> and <button2/> options. So you can access their functions on smaller screens.
If Sidebar is set to False and no items / components are added to the Header, Sidebar will not show up at all.